index.bs

```diff
@@ -14186,7 +14186,8 @@ the special value “missing”, which represents a missing optional argument.
         |callResult| and jump to the step labeled <a href="#call-user-object-operation-return"><i>return</i></a>.
     1.  Set |completion| to the result of [=converted to an IDL value|converting=]
         |callResult|.\[[Value]] to an IDL value of the same type as the operation's
```

Comment:
While here, maybe fix this to not say "operation's" for all three, and instead reference _callable_'s return type where appropriate?

Reply:
Done.